src/shared/helpers.js

```diff
@@ -18,3 +18,14 @@ export default {
 function isNumber(value) {
   return typeof value === "number" && isFinite(value);
 }
+
+export function cssObjectToString(style) {
+  return Object.entries(style)
+    .map(
+      ([k, v]) =>
+        `${k}{${Object.entries(v)
+          .map(([k, v]) => `${k}:${v}`)
+          .join(";\n")}}`
```

Comment:
Do we need a new line here? doesn't it just create additional bytes that aren't required?